byway/src/main/webapp/home.js

```diff
@@ -1,3 +1,13 @@
+/* global google, configureTripKeyForPath, setLogoutLink*/
+/* exported placesService*/
+
+let placesService;
+
+if(!!window.performance && window.performance.navigation.type == 2)
+{
+    window.location.reload();
+}
```

Comment:
If the user navigates to the page via the back button, this reloads the page. I did this because if user created a new trip and then clicked the back button to go back to home page, I wanted that new trip to appear on their list of trips

Reply:
Please use the named constant `PerformanceNavigation.TYPE_BACK_FORWARD` rather than a numeric constant/magic number.


Did you run into problems with this, it seems very odd to me that hitting 'back' would ot result in loading content on the previous page again.

Reply:
Yeah i did have problems. Without this, when I hit the back button, it wouldn't add the new trip until I refreshed the page.